### PR TITLE
kill multiple accumulators at once

### PR DESCRIPTION
**Pull request by tranma**
this reduces my small example from ~2s to 1s
@amosr

### CODE REVIEW

File: src/Icicle/Avalanche/Statement/Simp/Dead.hs [hunk 1]
```diff
@@ -40,70 +44,71 @@ instance Eq n => Monoid (Usage n) where
   , usageExp = Set.union (usageExp a) (usageExp b)
   }
```
Comment (amosr) [inline src/Icicle/Avalanche/Statement/Simp/Dead.hs]: could perhaps put kill as a field in here?

Comment (tranma) [inline src/Icicle/Avalanche/Statement/Simp/Dead.hs]: I tried that as well, but thought it's nicer on its own (maybe not)

Comment (amosr) [inline src/Icicle/Avalanche/Statement/Simp/Dead.hs]: no - you are right actually. I just saw that it doesn't thread through quite the same way after I commented

**Comment by amosr** [comment]
looks good :+1:

**Pull request merged**